README.md

```diff
@@ -1,25 +1,108 @@
-#Location Trivia Objects
+# Location Trivia Objects
 
-In the last lab you dealt with `NSDictionary` object representation of locations. Now lets convert these to proper classes and add in trivia items!
+## Objectives
+
+1. Create custom class files.
+2. Create, access, and set properties.
+3. Compose initializers.
+4. Write methods that interact with data stored in the instances of a custom class.
 
 ## Instructions
 
- 1. First create a new `FISLocation` class, add in the properties from the `NSDictionary`. So name, latitude and longitude
- 2. Move the `shortenLocationName` method to an instance method for your `FISLocation` class. Do you need the `NSDictionary` location argument anymore?
- 3. Create an initializer that takes in the three properties and assigns them correctly
- 4. Modify the `getLocationNamesWithLocations` method to take an `NSArray` of `FISLocation` objects and just return their names
- 5. Move the `verifyLocation` method to be an instance method for `FISLocation` and still return `YES` or `NO` whether the `FISLocation` has all of it's properties filled out
- 6. Create a designeated init method for FISLocation.
- 6. Modify the `searchForLocationName:inLocations:` method to search an `NSArray` of `FISLocation` objects.
- 7. Create a new class called `FISTrivia`. This class should have the following properties:
+In the last *Location Trivia* lab you dealt with a representation of locations using `NSDictionary`. Now, increase the power of your programming by storing this information in custom classes instead!
+
+*  Fork and clone this lab.
+
+*  Open the `locationTrivia-objects.xcworkspace` file.
+
+### I. Create the Custom Classes
+
+Create the class files for `FISLocation` that inherits from `NSObject`. This class should have four properties:
+
+  * an `NSString` called `name`,
+  * a `CGFloat` called `latitude`,
+  * a `CGFloat` called `longitude`, and
+  * an `NSMutableArray` called `trivia`.
+
+Create the class files for `FISTrivium` that inherits from `NSObject`. This class should have two properties:
+
+  * an `NSString` called `content`, and
+  * an `NSUInteger` called `likes`.
+
+On the `FISAppDelegate` class add a public property:
+
+  * an `NSMutableArray` called `locations`.
+
+### II. Declare Methods and Define Initial Implementations
+
+On the `FISAppDelegate` class, declare three methods:
+
+  * `allLocationNames` that takes no arguments and returns an `NSArray`,
+  * `locationWithName:` that takes one `NSString` argument called `name` and returns a `FISLocation` object, and
+  * `locationsNearLatitude:longitude:margin:` that takes three `CGFloat` arguments called `latitude`, `longitude`, and `margin`, and returns an `NSArray`.
+
+Define these methods to return `nil`.
+
+On the `FISLocation` class, declare five methods:
+
+  * `init` — the default initializer that will be overridden,
```

Comment:
@jmburges what's your opinion on this? I remember learning this as good practice but I can't explicitly say from where. Could've just been an opinion I found on the internet.